sup.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: org.falcao.sup
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: /app/sup
+finish-args:
+  # Required permissions for Electron apps
+  - --share=network
+  - --filesystem=home
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  # Environment variable to disable sandboxing
+  - --env=CHROME_DEVEL_SANDBOX=/app/chrome-sandbox
+  - --env=ELECTRON_DISABLE_SANDBOX=true
+
+modules:
+- name: sup
+  buildsystem: simple
+  build-commands:
+    # Extract the AppImage and copy its contents to /app
+    - ./Sup-1.0.0.AppImage --appimage-extract
+    - cp -r squashfs-root/* /app/
+    # Ensure the main executable and chrome-sandbox permissions are set correctly
+    - chmod +x /app/sup
+    - chmod 4755 /app/chrome-sandbox || true
+  sources:
+    # Include the AppImage file as a source
+    - type: file
+      path: Sup-1.0.0.AppImage
```

Comment:
this file doesn't exist. And I will add it should NEVER be added to this submission.

Reply:
why not building from source.